tests/FakeItEasy.Tests/Expressions/ArgumentConstraints/AggregateArgumentConstraintTests.cs

```diff
@@ -14,7 +14,7 @@ public AggregateArgumentConstraintTests()
             this.ConstraintField = new AggregateArgumentConstraint(new[] { new EqualityArgumentConstraint("foo"), new EqualityArgumentConstraint("bar") });
         }
 
-        public interface ITypeWithPaldkf

```

Comment:
I can put it back, if you prefer.

But if you're asking where this came from, 8fce4e74354ab120eaa20667fa207b5556d4c447.